projects/cashmere/src/lib/modal/modal-window.component.spec.ts

```diff
@@ -42,42 +42,6 @@ describe('hc-modal-window', () => {
         expect(windowComponent._ignoreOverlayClick).toBeTruthy();
     });
 
-    describe('should change size', () => {
```

Comment:
Bummer to rip out tests that could probably be fixed. But I'm not enough of a stickler to let this prevent a merge

Reply:
Ugh, I know I thought that same thing - these tests are still relevant and probably shouldn't be removed.  They would just have to be engineered another way.  Maybe you and I can try to figure out a fix to get them back in this afternoon in our 1:1.

Reply:
Had to start a new set of unit tests for the modal service which didn't exist previously, but allowed me to test the size and resizable options.  Probably some additional coverage could be added on that one, but at least it's a start - and we haven't lost the coverage we had before.